app/src/main/java/com/teamagam/gimelgimel/app/common/base/adapters/BaseRecyclerArrayAdapter.java

```diff
@@ -133,7 +150,7 @@ public void onChanged(int position, int count) {
 
     @Override
     public boolean areContentsTheSame(D oldItem, D newItem) {
-      return oldItem.getId().equals(newItem.getId());
+      return false;
```

Comment:
So it'll always update the view in-case we update an item (we didn't do that before)